comment/__init__.py

```diff
@@ -1 +1,8 @@
 default_app_config = 'comment.apps.CommentConfig'
+
+__version__ = "2.0.0"
```

Comment:
Why didn't you just read VERSION file here instead of using a static string. I would prefer to change VERSION file on each releasing rather than changing the __init__ file.

Reply:
I had not seen the manifest file when writing this. I thought we are not shipping with the package to the end users.

Now that you put it this way, this solution seems really silly :(

I actually looked for this the source code of *pip*.

I don't know why they do it like this? Maybe because they don't want users to download one extra file or something else?

Shall I revert the change?

Reply:
Yes please.